samples/bot-adaptive-card-actions/csharp/README.md

```diff
@@ -0,0 +1,128 @@
+     - `{{MicrosoftAppId}}` - Generated from Step 1 is the application app id
+     - `{{MicrosoftAppPassword}}` - Generated from Step 1, also referred to as Client secret
+   
+   - In a terminal, navigate to `samples/bot-adaptivecard-actions/csharp`
+
+     change into project folder
+      ```bash
+      cd # AdaptiveCardActions
+      ```
+   - From a terminal
+     ```bash
+     # run the bot
+     dotnet run
+     ```
+
+   - Or from Visual Studio
+     - Launch Visual Studio
+     - File -> Open -> Project/Solution
+     - Navigate to `AdaptiveCardActions` folder
+     - Select `AdaptiveCardActions.csproj` file
+     - Press `F5` to run the project
+     
+5. Setup Manifest for Teams
+- __*This step is specific to Teams.*__
+    - **Edit** the `manifest.json` contained in the ./AppPackage folder to replace your Microsoft App Id (that was created when you registered your app registration earlier) *everywhere* you see the place holder string `{{Microsoft-App-Id}}` (depending on the scenario the Microsoft App Id may occur multiple times in the `manifest.json`)
+    - **Edit** the `manifest.json` for `validDomains` and replace `{{Domain-Name}}` with base Url of your domain. E.g. if you are using ngrok it would be `https://1234.ngrok.io` then your domain-name will be `1234.ngrok.io`.
+    - **Zip** up the contents of the `AppPackage` folder to create a `manifest.zip` (Make sure that zip file does not contains any subfolder otherwise you will get error while uploading your .zip package)
+
+- Upload the manifest.zip to Teams (in the Apps view click "Upload a custom app")
+   - Go to Microsoft Teams. From the lower left corner, select Apps
+   - From the lower left corner, choose Upload a custom App
+   - Go to your project directory, the ./AppPackage folder, select the zip folder, and choose Open.
+   - Select Add in the pop-up dialog box. Your app is uploaded to Teams.
+
+
+## Running the sample
+
+![App Setup](AdaptiveCardActions/Images/1.AddApp.png)
+
+![Welcome](AdaptiveCardActions/Images/2.Welcome.png)
+
+![Runnning Sample](AdaptiveCardActions/Images/3.AdaptiveCard_Actions.png)
+
+![Runnning Sample](AdaptiveCardActions/Images/4.ActionSubmit.png)
+
+![Runnning Sample](AdaptiveCardActions/Images/5.ActionShowCard.png)
+
+![Runnning Sample](AdaptiveCardActions/Images/6.Togglevisible.png)
+
+![Runnning Sample](AdaptiveCardActions/Images/7.ToggleVisibleOnClick.png)
+
+![Runnning Sample](AdaptiveCardActions/Images/8.SuggestedActions.png)
+
+![Runnning Sample](AdaptiveCardActions/Images/9.RedColor.png)
+
+![Runnning Sample](AdaptiveCardActions/Images/10.BlueColor.png)
+
+![Runnning Sample](AdaptiveCardActions/Images/11.YellowColor.png)
+
+## Deploy the bot to Azure
+
+To learn more about deploying a bot to Azure, see [Deploy your bot to Azure](https://aka.ms/azuredeployment) for a complete list of deployment instructions.
+
+## Further reading
+
+- [Bot Framework Documentation](https://docs.botframework.com)
```

Comment:
We already discussed this multiple time Tomorrow but still you are adding these links.

Add the Doc that you have followed (Microsoft Doc link)

Reply:
fixed